src/components/actionable-item/ActionableItem.tsx

```diff
@@ -1,25 +1,36 @@
 import * as classNames from 'classnames';
 import * as React from 'react';
+import * as _ from 'underscore';
+import {callIfDefined} from '../../utils/FalsyValuesUtils';
 import {IItemBoxProps} from '../itemBox/ItemBox';
 import {ListBox} from '../listBox/ListBox';
 import {PopoverConnected} from '../popover/PopoverConnected';
 import {Svg} from '../svg/Svg';
 import {actionableItemContainer, actionableItemContent, actionableItemDots, actionableItemDotsWidthWithPadding} from './styles/ActionableItem.scss';
 
-export interface IActionableItemProps extends React.HTMLAttributes<HTMLDivElement> {
+export interface IActionableItemProps {
     id: string;
+    onItemClick?: (e?: React.MouseEvent<HTMLDivElement>) => any;

```

Comment:
you can return void

Reply:
thanks, you're right, both are equivalent in this case i'd say